nodeup/pkg/model/kubectl.go

```diff
@@ -104,20 +103,9 @@ func (b *KubectlBuilder) Build(c *fi.ModelBuilderContext) error {
 
 // findKubeconfigUser finds the default user for whom we should create a kubeconfig
 func (b *KubectlBuilder) findKubeconfigUser() (*fi.User, *fi.Group, error) {
-	var users []string
-	switch b.Distribution {
-	case distributions.DistributionDebian9, distributions.DistributionDebian10:
-		users = []string{"admin", "root"}
-	case distributions.DistributionUbuntu1604, distributions.DistributionUbuntu1804, distributions.DistributionUbuntu2004, distributions.DistributionUbuntu2010:
-		users = []string{"ubuntu"}
-	case distributions.DistributionCentos7, distributions.DistributionCentos8:
-		users = []string{"centos"}
-	case distributions.DistributionAmazonLinux2, distributions.DistributionRhel7, distributions.DistributionRhel8:
-		users = []string{"ec2-user"}
-	case distributions.DistributionFlatcar:
-		users = []string{"core"}
-	default:
-		klog.Warningf("Unknown distro; won't write kubeconfig to homedir %s", b.Distribution)
+	users, err := b.Distribution.DefaultUsers()
+	if err != nil {
+		klog.Warningf("won't write kubeconfig to homedir for distribution %s: %v", b.Distribution, err)
```

Comment:
Will we start wrapping inner errors via the `%w` verb?

Reply:
I think we should yes, but annoyingly klog doesn't yet recognize it (last I checked!), so we have to use %v.

So yes for fmt.Errorf, not yet for klog